lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -60,21 +60,20 @@ class ItemListPresenter {
 
     lazy private(set) var sortingButtonObserver: AnyObserver<Void> = {
         return Binder(self) { target, _ in
-            target.view?.displayOptionSheet(buttons: [
-                OptionSheetButtonConfiguration(
```

Comment:
The rename of `OptionSheetButtonConfiguration` to `AlertActionButtonConfiguration` will require corresponding updates in the ItemListPresenter test specs: https://dashboard.buddybuild.com/apps/5a0ddb736e19370001034f85/build/5adfb42220c231000196179f